Formula/php-cs-fixer.rb

```diff
@@ -12,7 +13,12 @@ class PhpCsFixer < Formula
   depends_on "php"
```

Comment:
Don't we want these to depend on an earlier version of `php`?

Reply:
Yes, this should be `php@8.0` as soon as there's a dedicated formula for it. The audit script won't let me do that as long as `php@8.0` is just an alias for `php`.

Reply:
Why not 7.4 until then?

Reply:
Now that we have a `php@8.0` formula, I've updated the PR to use it.